docs/reference/connector/apis/update-connector-pipeline-api.asciidoc

```diff
@@ -9,6 +9,10 @@ preview::[]
 
 Updates the `pipeline` configuration of a connector.
 
+When you create a new connector, the configuration of an <<ingest-pipeline-search-details-generic-reference, ingest pipeline>> is populated with default settings.
+
+include::{es-repo-dir}/ingest/search-ingest-pipelines.asciidoc[]
```

Comment:
@jedrazb I'm not sure if including the entire doc page makes sense here, because it will just render that whole doc.

Consider using the link syntax mentioned [above](https://github.com/elastic/elasticsearch/pull/103450/files/f0baa2111d5048412f899b5d0b157cdc90287f51..4ff64c14d186898050cfbb57594c9f0dae5578cf#r1426957623)

Reply:
I though it's needed to resolve the hyperlink referemce. I will delete this and check the preview ;)